docs/user/README.md

```diff
@@ -0,0 +1,193 @@
+
+
+### Domain model
+
+All business data are completely stored in ETCD, using [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+
+The data model consists of two CRDs:
+
+* `Runtime`
+
+    Cluster data are stored in instances of the  CRD. 
+
+* `RuntimeKubeconfig`
+
+    Kubeconfig related metadata are stored in entities of the  CRD.
+
+The structure and the purpose of the different fields in these resources are documented in the CRD files:
+
+https://github.com/kyma-project/kyma-infrastructure-manager/tree/main/config/crd/bases
+
+
+### State Machine
+The reconciliation process is implemented by using a [state machine pattern](https://en.wikipedia.org/wiki/Finite-state_machine). Each state represents a reconciliation step. Based on the return value of a step, the transition to a next steps happens.
+
+
+### Sub-Components
+During the implementation of paritcular KIM features were sub-components introduced to address the feature-specifc requirements:
+
+* KIM Snatch
+
+    This is a Kyma module which belongs to KIM and was introduced with the support for multiple worker pools. It's a mandatory module: KLM installs it on all Kyma runtimes automatically, customers cannot avoid this or remove the module. Purpose of KIM Snatch is to assign Kyma workloads to the Kyma worker pool. This is needed to avoid that Kyma workloads will be assigned to worker pools created by customers.
+
+    More information are available in the KIM Snatch repository: https://github.com/kyma-project/kim-snatch/tree/main/docs/user
+
+* Gardener Syncer
+
+    This Kubernetes Cronjob synchronizes Seed cluster data from the Gardener cluster to KCP. This data is used by KEB for input validation of customers (primarily for the "Shoot and Seed in same region" feature to detect if a Seed cluster exists in a particular Shoot region).
+
+    The seed data are stored in a config map in KCP.
+
+    More details includes the documentation of the Gardener Syncer: https://github.com/kyma-project/gardener-syncer/blob/main/README.md
+
+
+### ADR
+Please see the ADR (Architectural Decision Record) of KIM tp get more insights about its [software architecture](../adr/001-provisioning.md).
+
+
+## Quality Requirements
+
+Non-technical requirements for KIM are:
+
+* Performance
+
+    Goal was to be able to process +/- 5.000 Runtime instances with KIM. A load and performance test was implemented which confirms the achievement of this goal.
+
+* Extensibility
+
+    Selected software patterns are used to ensure an easy extensibility of KIM:
+    
+    *  state machine pattern is used for process flow control
+    *  chain of responsibility pattern is used for rendering the Shoot definitions
+
+*  Reliability
+
+    Uptime is crucial for KIM. Monitoring is detecting throughput degradation and long-running reconciliation processes. Alerting rule are in place to notify the team about SLA violations.
+
```

Comment:
```suggestion
*  Reliability: Since uptime is crucial for KIM, monitoring tools are employed to detect throughput degradation and long-running reconciliation processes, with alerting rules to notify about any SLA violations.
```